pringles/backends/statics/basic.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html class="basic-page">
+
+<head>
+  <meta charset="UTF-8">
+  <title>DEVS Diagrammer</title>
+  <meta name="viewport" content="width=device-width, user-scalable=no">
+
+  <link rel="stylesheet" href="{{ url_prefix }}/_static/css/style.css" />
+
+  <script src="{{ url_prefix }}/_static/vendors/js/createjs/createjs-2013.12.12.min.js"></script>
```

Comment:
Related to other comment about hardcoded "localhost:<aPort>":

Doesn't it work if you just put /_static/... without the prefix? I think if you do that it resolves the url as being from the same domain

Reply:
The prefix is needed. The web-server that serve this statics is listening a port different from the one jupyter is binded on. So if you don't add the`{{ url_prefix }}`, it will try to fetch each static from the jupyter URL.